spec/features/issue_trash_recover_spec.rb

```diff
@@ -15,6 +15,8 @@
     visit "/trash"
     click_link 'Recover'
 
+    sleep(1)
```

Comment:
I think we should get to the bottom of this, adding a spurious 1sec delay is only making our life worse in the long run.